hack/jenkins/job-configs/kubernetes-upgrades.yaml

```diff
@@ -109,72 +117,120 @@
     jobs:
         - 'kubernetes-upgrade-{provider}-{version-old}-{version-new}'
         - 'kubernetes-upgrade-{provider}-{version-old}-{version-new}-step1-deploy':
-            branch: '{branch-old}'
+            runner: '{runner-old}'
+            job-env: |
+                export E2E_TEST="false"
+                export E2E_DOWN="false"
         - 'kubernetes-upgrade-{provider}-{version-old}-{version-new}-{step}':
             step: 'step2-kubectl-e2e-new'
-            branch: '{branch-new}'
             description: 'Run {version-new} kubectl tests against the cluster running {version-old}.'
+            job-env: |
+                export E2E_OPT="--check_version_skew=false"
+                export E2E_UP="false"
+                export E2E_DOWN="false"
+                export GINKGO_TEST_ARGS="--ginkgo.focus=Kubectl"
         - 'kubernetes-upgrade-{provider}-{version-old}-{version-new}-{step}':
             step: 'step3-upgrade-master'
-            branch: '{branch-new}'
             description: 'Upgrade the master from {version-old} to {version-new}.'
+            job-env: |
+                export E2E_OPT="--check_version_skew=false"
+                export E2E_UP="false"
+                export E2E_DOWN="false"
+                export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Upgrade\].*upgrade-master"
```

Comment:
This isn't going to work for v1.1; see https://github.com/kubernetes/kubernetes/blob/release-1.1/hack/jenkins/e2e.sh#L168